ipa-core/tests/common/mod.rs

```diff
@@ -93,12 +93,18 @@ pub trait CommandExt {
 }
 
 impl CommandExt for Command {
+    // Have the `silent` function return self and comment out the
+    // rest of the function to see printing from tests that run all
+    // the binaries. e.g. when running: `cargo test --test compact_gate --lib
+    // compact_gate_cap_8_no_window_semi_honest -p ipa-core --no-default-features
+    // --features "cli web-app real-world-infra test-fixture compact-gate"`
     fn silent(&mut self) -> &mut Self {
         if std::env::var("VERBOSE").ok().is_none() {
             self.arg("--quiet")
         } else {
             self.arg("-vv")
         }
+        // return self;
```

Comment:
Do you want to leave this in, or was this leftover from debugging?

Reply:
Yes, it is left from debugging, but I think it is worth leaving it together with the added documentation for this function.  This was a trick I learned from Alex for debugging tests that run the compiled binaries, but I think we should somewhere makes how to do this better known.  Probably not just here but in some developer documentation.